BundleClient/app/src/main/java/net/discdd/datastore/providers/MessageProvider.java

```diff
@@ -12,6 +12,7 @@
 import android.os.Binder;
 
 import java.io.File;
+import java.nio.file.Path;
```

Comment:
Is this import required? Can you please check and remove it if it is not needed?

Reply:
removed.